sdk/core/core-http/CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## 1.2.0 (2020-10-19)
 
 - Explicitly set `manual` redirect handling for node fetch. And fixing redirectPipeline [PR 11863](https://github.com/Azure/azure-sdk-for-js/pull/11863)
-- Add support for multiple error response codes.[PR 11841](https://github.com/Azure/azure-sdk-for-js/)
+- Add support for multiple error response codes. [PR 11841](https://github.com/Azure/azure-sdk-for-js/)
```

Comment:
these notes should be in a newer section right or we need to fix the release date for 1.2.0 above, right?

Reply:
The date will be fixed in #12258